src/app/Models/Traits/SpatieTranslatable/HasTranslations.php

```diff
@@ -80,7 +80,13 @@ public static function create(array $attributes = [])
         // do the actual saving
         foreach ($attributes as $attribute => $value) {
             if ($model->isTranslatableAttribute($attribute)) { // the attribute is translatable
-                $model->setTranslation($attribute, $locale, $value);
+                if (config('backpack.crud.override_available_locales_on_create')) {
```

Comment:
I don't think the config name represents what we would like to do here. 
We are "overwriting" anything, as there is nothing in other locales at the moment. 

So something like `backpack.crud.fill_missing_locales` would be more descriptive/accurate I think.